lib/task-data/schemas/install-os-general.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "rackhd/schemas/v1/task-schema",
```

Comment:
Thanks @brianparry, some local people still have same concern that the install-os schema structure is a little complex. I will improve this later.